core/iwasm/aot/aot_runtime.c

```diff
@@ -1164,6 +1165,15 @@ aot_instantiate(AOTModule *module, AOTModuleInstance *parent,
     }
 #endif
 
+    common = &((AOTModuleInstanceExtra *)module_inst->e)->common;
+    common->data_dropped = bh_bitmap_new(0, module->mem_init_data_count);
+    common->elem_dropped = bh_bitmap_new(0, module->table_init_data_count);
```

Comment:
Could we allocate memory only when mem_init_data_count/mem_init_data_count is larger than 0? Or there may be a warning:
https://github.com/bytecodealliance/wasm-micro-runtime/blob/main/core/iwasm/common/wasm_memory.c#L245

Reply:
done